ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -10,4 +10,5 @@
 	<filter root="/var/workflow/models/hclecomm" mode="merge"/>
 	<filter root="/content/forms/af/hclecomm" mode="merge"/>
 	<filter root="/content/dam/formsanddocuments/hclecomm" mode="merge"/>
+	<filter root="/content/dam/hclecomm/en/resources/botpress" mode="merge"/>
```

Comment:
@VikramGoud1576 : we already have a filter at line 5  <filter root="/content/dam/hclecomm" mode="merge"/> , why do we need a separate filter ?

Reply:
removed the filter